spec/factories/items.rb

```diff
@@ -2,5 +2,6 @@
   factory :item do
     title "one"
     body "one body"
+    association :user
```

Comment:
`association` って、個人的には使わないのですけど `user` だけ書くのとどう違うんでしょうか？

Reply:
FactoryBot のページに書かれていました。動作は同じようです。
エイリアスで省略できるので、この記法を使うようにしたいと思います。

```ruby
factory :user, aliases: [:author, :commenter] do
  first_name { "John" }
  last_name { "Doe" }
  date_of_birth { 18.years.ago }
end

factory :post do
  author
  # instead of
  # association :author, factory: :user
  title { "How to read a book effectively" }
  body { "There are five steps involved." }
end

factory :comment do
  commenter
  # instead of
  # association :commenter, factory: :user
  body { "Great article!" }
end
```
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#aliases